x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authz/AuthorizationService.java

```diff
@@ -635,17 +635,33 @@ private void runRequestInterceptors(
             listener.onResponse(null);
```

Comment:
Don't need this branch any more I think?

Reply:
Right :) and it's gone.